src/app/group/group-list/group-list.component.ts

```diff
@@ -8,61 +10,106 @@ import { ActivatedRoute, Router } from '@angular/router';
+    { groupName: 'Mumbai', shareLink: 'tff-dfs-sdf', groupType: 'home', groupMember: [{ memberName: 'Priya' }, { memberName: 'Mayank' }, { memberName: 'Harish' }] }
   ];
 
   transactionList = [
-    { payer:'Priya', receiver:'Mayank',currency:'USD', amount:1000},
-    { payer:'Harish', receiver:'Mayank',currency:'USD', amount:2000},
-    { payer:'Nikita', receiver:'Priya',currency:'₹' , amount:3000}
+    { payer: 'Priya', receiver: 'Mayank', currency: 'USD', amount: 1000 },
+    { payer: 'Harish', receiver: 'Mayank', currency: 'USD', amount: 2000 },
+    { payer: 'Nikita', receiver: 'Priya', currency: '₹', amount: 3000 }
   ];
 
   expenseList = [
-    { expenseName:'Book', paidBy:'Mayank', lent:'Harish', paidAmount:'5000', lentAmount:'1666', currency:'₹'},
-    { expenseName:'Food', paidBy:'Priya', lent:'Nikita', paidAmount:'500', lentAmount:'166', currency:'₹'},
-    { expenseName:'Taxi', paidBy:'Harish', lent:'Nikita', paidAmount:'1000', lentAmount:'666', currency:'USD'}
+    { expenseName: 'Book', paidBy: 'Mayank', lent: 'Harish', paidAmount: '5000', lentAmount: '1666', currency: '₹' },
+    { expenseName: 'Food', paidBy: 'Priya', lent: 'Nikita', paidAmount: '500', lentAmount: '166', currency: '₹' },
+    { expenseName: 'Taxi', paidBy: 'Harish', lent: 'Nikita', paidAmount: '1000', lentAmount: '666', currency: 'USD' }
   ]
 
 
+
   // Constructor
-  constructor( private router: Router, private activatedRoute: ActivatedRoute){};
+  constructor(private router: Router,
+    private activatedRoute: ActivatedRoute,
+    private groupsService: GroupsService
+  ) {
+    this.getGroupList();
+  };
 
   // ngOnInit method
   ngOnInit(): void {
-    
+
     // get activatedRoute parameter using observable
-    this.activatedRoute.params.subscribe((param) =>{
+    this.activatedRoute.params.subscribe((param) => {
       this.getActivatedRouteParam = param['routerParam'];
-      if(this.getActivatedRouteParam === undefined ){
+      if (this.getActivatedRouteParam === undefined) {
         this.isGroupDetailActive = false;
       }
-      else{
+      else {
         this.isGroupDetailActive = true;
         this.groupNameDetail = this.getActivatedRouteParam;
       }
     })
   }
 
   // Group detail show
-  groupDetailShow(name: string, index: number): void{
+  groupDetailShow(name: string, index: number): void {
     this.getActivatedRouteParam = name;
-    this.router.navigate(['group','group-list',name]);
+    this.router.navigate(['group', 'group-list', name]);
   }
 
   // Navigate to edit form
-  navigateToEditForm(index: number): void{
-    this.router.navigate(['group','group-edit',index]);
+  navigateToEditForm(index: number): void {
+    this.router.navigate(['group', 'group-edit', index]);
   }
 
-  // Delete group
+  /**  Delete group function
+   * to delete group from list*/
```

Comment:
last */ should be in next line you must maintain the code consistency including space, comma, sentences everything